Tests/Functional/CompositeRepositoryTest.php

```diff
@@ -13,6 +13,8 @@
 
 class CompositeRepositoryTest extends PhpcrRepositoryTestCase
```

Comment:
Why extend the PhpcrTestCase here? CompositeRepository has nothing to do with PHPCR ...